### PR TITLE
readme clarification

### PR DESCRIPTION
**Pull request by dperegolise**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
@mechmerlin @ShadeDream I know you guys both worked on this codebase.

File: keyboards/doro67/multi/readme.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 65% custom keyboard made by 80ultraman/Alf/Backprop Studios with multiple layout support. Despite the layout options available, layout is dictated by the selected integrated plate.  
 
+**THIS IS THE KEYMAP DIRECTORY FOR NON-CHINA BUYERS**
```
Comment (noroadsleft) [inline keyboards/doro67/multi/readme.md]:
```suggestion
**THIS IS THE MULTI-LAYOUT PCB DIRECTORY FOR NON-CHINA BUYERS**
```

File: keyboards/doro67/regular/readme.md [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 This is not the PCB with RGB support. Do not flash RGB firmware for this board. 
 
+**THIS IS THE DEFAULT KEYMAP DIRECTORY FOR CHINA BUYERS**
```
Comment (noroadsleft) [inline keyboards/doro67/regular/readme.md]:
```suggestion
**THIS IS THE DIRECTORY FOR CHINA BUYERS**
```

File: keyboards/doro67/rgb/readme.md [hunk 3]
```diff
@@ -4,6 +4,12 @@
 
 Flashing the regular PCB firmware on this board will work, but will disable RGB lighting. 
 
+**THIS IS THE RGB KEYMAP DIRECTORY FOR NON-CHINA BUYERS**
```
Comment (noroadsleft) [inline keyboards/doro67/rgb/readme.md]:
```suggestion
**THIS IS THE RGB PCB DIRECTORY FOR NON-CHINA BUYERS**
```

**Comment by dperegolise** [comment]
I like it, applied

File: keyboards/doro67/multi/readme.md (hunk 1 above)
Comment (mechmerlin) [reply keyboards/doro67/multi/readme.md]: Non China buyers sounds like if you're not Chinese you can use this lol. 

Maybe

```suggestion
**MULTI-LAYOUT PCB (AVAILABILITY: CHINA + INTERNATIONAL GB)**
```

File: keyboards/doro67/regular/readme.md (hunk 2 above)
Comment (mechmerlin) [reply keyboards/doro67/regular/readme.md]:
```suggestion
**REGULAR PCB (AVAILABILITY: CHINA GB ONLY)**
```

File: keyboards/doro67/rgb/readme.md (hunk 3 above)
Comment (mechmerlin) [reply keyboards/doro67/rgb/readme.md]:
```suggestion
**RGB PCB (AVAILABILITY: CHINA + INTERNATIONAL GB)**
```

**Comment by dperegolise** [comment]
Added @mechmerlin suggested verbiage
Thanks!

**Comment by ShadeDream** [comment]
Changes look good to me. 👍

**Pull request merged**